src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -297,7 +362,7 @@ public void run() {
                 // @checkstyle MagicNumber (1 line)
                 final byte[] buffer = new byte[4096];
                 for (int count = input.read(buffer); count != -1;
-                    count = input.read(buffer)) {
+                     count = input.read(buffer)) {
```

Comment:
@dmzaytsev this spaces added by supposed codestyle config for IDEA from @yegor256 and caused by limited to 80 chars maximum line lenght (because 'for' statement is too long for 80 chars). Also this style did not cause warnings in Qulice.
I dont think that manual code reformatting is a good and productive way to create clean code.
I will be very appreciate to you if your share IDE config with this check.

Regarding to another comments i will fix them.